python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java

```diff
@@ -542,20 +542,21 @@ public void modules_whose_name_differ_by_capitalization_only() {
     // Python 2, import SocketServer
     setPythonVersions(PythonVersionUtils.fromString("2.7"));
     Map<String, Symbol> socketServer = symbolsForModule("SocketServer");
-    assertThat(socketServer).isNotEmpty();
-    SymbolImpl baseServer = (SymbolImpl) socketServer.get("BaseServer");
-    assertThat(baseServer.validForPythonVersions()).containsExactlyInAnyOrder("27");
+    assertThat(socketServer).isEmpty();
 
     // Python 2, import socketserver
     // SONARPY-1359: socketserver is missing from the standard library stubs in Python 2, but it is indirectly serialized through the stubs serialized by the mypy importer
     socketServer = symbolsForModule("socketserver");
     assertThat(socketServer).isNotEmpty();
+    SymbolImpl baseServer = (SymbolImpl) socketServer.get("BaseServer");
+    assertThat(baseServer.validForPythonVersions()).isEmpty();
```

Comment:
Not 100% sure about this one as we do find the symbol but it is not valid for 27